### PR TITLE
Stroke width aesthetic for geom_point

### PR DESCRIPTION
**Pull request by faizan-khan-iit**
This PR covers issue #127

### CODE REVIEW

**Comment by faizan-khan-iit** [comment]
Some comments here:

With the current implementation, the animint and the ggplot will look a little different.
As quoted in the issue #127 -

> size and stroke are additive so a point with `size = 5` and `stroke = 5` will have a diameter of 10mm

But while using SVG, strokes are aligned to the center of each shape’s edge. That is, the stroke is neither fully inside nor outside the object, but is both half in and half out. So `size=5` and `stroke=5` gives a point with diameter 7.5 mm, with a stroke of width 5mm.

Do we need to fix that?

File: inst/htmljs/animint.js [hunk 1]
```diff
@@ -1080,7 +1080,19 @@ var animint = function (to_select, json_file) {
       }
       return size;
     };
-
+    
+    // stroke_width for geom_point
+    var stroke_width = 0.5;  // by default ggplot2 has 0.5
```
Comment (tdhock) [inline inst/htmljs/animint.js]: the previous default in animints seems to be stroke_width=1 so can we please keep that?
screenshot-stroke-width

File: tests/testthat/test-renderer4-geom-point-stroke.R [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+p2 <- ggplot(mtcars, aes(wt, mpg, stroke=cyl)) +
+  geom_point(shape = 21, colour = "black", fill = "white", size = 5)
+
+viz <- list(p1=p1, p2=p2)
+info <- animint2HTML(viz)
+
+test_that("points are rendered with stroke-width", {
+  point_nodes <-
+    getNodeSet(info$html, '//g[@class="geom1_point_p1"]//circle')
```
Comment (tdhock) [inline tests/testthat/test-renderer4-geom-point-stroke.R]: please use `getStyleValue(, , "stroke-width")`

**Comment by tdhock** [comment]
good job on this first implementation!

It would be nice if the animint looked exactly like the ggplot, but in this situation it seems too complicated to achieve. so no need to fix that.

for the tests, please add `expect_equal(stroke_width_from_html, stroke_from_R_code)`

**Comment by faizan-khan-iit** [comment]
@tdhock Thanks for the tip! Using that initially would have saved me some effort..

**Comment by tdhock** [comment]
looks good to me +1

go ahead and edit DESCRIPTION/NEWS, then merge with master

**Pull request merged**